src/amo/components/Link.js

```diff
@@ -0,0 +1,27 @@
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { Link } from 'react-router';
+
+
+export class InternalLinkBase extends React.Component {
```

Comment:
I don't mind calling it `LinkBase`, no reason not to and as mentioned before, I don't think it needs to be prefixed with `Internal` actually.